### ISSUE TITLE
Signature of the WebGPURenderer

### ISSUE THREAD

**Issue by wcandillon** [three-types/three-ts-types#1165]
When using the WebGPURenderer, the following two properties don't appear to be in the type:
* `context`
* `device`

Below is an example
```ts
      renderer = new THREE.WebGPURenderer({
        antialias: true,
        canvas: context.canvas,
        // @ts-expect-error
        context,
        // @ts-expect-error
        device,
      });
```

**Comment by Methuselah96** [comment]
Should be fixed by #1173, which should hopefully get released with `@types/three@0.168.0` at the end of the month

**Comment by wcandillon** [comment]
Thank you very much @Methuselah96 and a big thank you the great work you do here 🙌🏼
